docs/guide/docs/integrating/existing-backend.md

```diff
@@ -1,26 +1,27 @@
 ---
 id: existing-backend
-title: Connecting your bot with your existing backend
+title: Connecting An Existing Backend
```

Comment:
I wonder if this is confusing. It seems like we are referring to connecting a backend for the specific purpose of doing Auth. However, we could connect an existing backend to retrieve information or store information. Should the title be different?